roundwared/recording_collection.py

```diff
@@ -184,6 +184,18 @@ def has_played(self):
         """
         return (len(self.banned_proximity) > 0 and len(self.banned_timeout) > 0)
 
+    def order_assets(self):
+        """
+        Order assets in playlist by ordering method set in project config
+        """
+        logger.debug("Ordering assets by: %s" % self.ordering)
+        if self.ordering == 'random':
+            self.playlist_timed = order_assets_randomly(self.playlist_timed)
```

Comment:
This is only changing playlist_timed, not playlist_proximity. You need the function call to work like this:
```
    self.playlist_timed = self.order_assets(self.playlist_timed)
```
and
```
    self.playlist_proximity = self.order_assets(self.playlist_proximity)
```